lib/far_mar/market.rb

```diff
@@ -0,0 +1,37 @@
+class FarMar::Market < FarMar::Base
+  attr_accessor :id,
+                :name,
+                :address,
+                :city,
+                :county,
+                :state,
+                :zip
+
+  def initialize(attrs)
+    @id      = attrs[0].to_i
+    @name    = attrs[1]
+    @address = attrs[2]
+    @city    = attrs[3]
+    @county  = attrs[4]
+    @state   = attrs[5]
+    @zip     = attrs[6]
+  end
+
+  def self.path
+    "support/markets.csv"
+  end
+
+  def self.find_by_state(state_name)
+    row = read_file.find { |line| line[5] == state_name }
+    new(row)
```

Comment:
Just for consistency, I would would use the `all` method here instead of `read_file`, this will save the step of having to instantiate a new object.